common/math_util.hpp

```diff
@@ -0,0 +1,79 @@
+#pragma once
+
+namespace ILLIXR {
+	class math_util {
+		public:
+
+		static void projection_fov(Eigen::Matrix4f* result,
```

Comment:
I know that this is the same logic as the ks algebra library, but it would be nice to have a reference for the math.

Reply:
Will do.